Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst

```diff
@@ -0,0 +1,129 @@
+
+The drives are power off, and the AuxTel Control cabinet is locked from the outside. 
+The AuxTel is under LOTO and is not able to move by any controls.
+
+.. _AuxTel-LOTO-procedure-Procedure-Steps:
+
+Procedure Steps
+^^^^^^^^^^^^^^^
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+To put the AuxTel under LOTO, follow these steps:
+
+#. Post in the Slack channels `#summit-announce <https://lsstc.slack.com/archives/C01P41NUR1R>`__ and `#summit-auxtel <https://lsstc.slack.com/archives/C01K4M6R4AH>`__ that you are going to put AuxTel under LOTO. 
+
+#. Announce your intentions on the radio (channel 3).
+
+#. Verify that no one is working in the dome, on the ground or second floors. 
+
+#. Identify the AT Control cabinet that you need to lock out, shown in the figure. 
+
+.. figure:: ./_static/AT-control-cabinet.png
+    :name: AT-control-cabinet
+
+    AT Control cabinet, inside, and EATON brakers.
+
+#. Inside the AT Control cabinet, identify the three (3) EATON brakers also shown in the figure, tagged from left to right. 
+
+   a. DRV #1 AZ AZ - The two drives for Azimuth motion.
+
+   b. DRV #2 EL M3 - The drives for elevation and tertiary mirror motion.
+
+   c. DRV #3 NR1 NR2 - Motors for the nasmyth rotators 1 and 2. 
+
+#. Switch off all three brakers by flipping the switch down. 
+
+#. Close the cabinet, and secure your lock and tag in the loop underneath the cabinet door handle. 
+
+#. Verify that the AuxTel will not move intentionally by sending a command with the Engineering User Interface (EUI) on the machine located on the first floor. 
+
+#. AuxTel is under LOTO, and work may begin. 
+
+
+.. _AuxTel-LOTO-procedure-Troubleshooting:
+
+Troubleshooting
+^^^^^^^^^^^^^^^
+
+.. This section should include troubleshooting information. Information in this section should be strictly related to this procedure.
+
+.. If there is no content for this section, remove the indentation on the following line instead of deleting this sub-section.
+
+No troubleshooting information is applicable to this procedure.
+In the NEARLY IMPOSSIBLE event that the telescope moves after turning off all three brakers, stop work immediately and notifty the day crew, including Jacques Sebag, Mario Rivera, and the electronics team. 
```

Comment:
Typo: "breakers". But I also have to ask, why do we say it is nearly impossible but not impossible? A mechanical event or something?

Reply:
Removed "nearly" and just left impossible. But, sometimes the impossible happens, so I left it to say, if you think the LOTO is complete and the telescope still moves, we in trouble. 👍